packages/xarc-react-router/src/browser/react-router-browser.tsx

```diff
@@ -20,10 +20,10 @@ export function reactRouterFeature(options: ReactRouterFeatureOptions): SubAppFe
       let history: any;
       let TheRouter: any;
 
-      if (!options.history) {
+      if (options.history === false) {
         TheRouter = BrowserRouter;
       } else {
-        history = options.history === true ? createBrowserHistory() : options.history;
+        history = options.history === true ? reactRouterHistory : (options.hasOwnProperty("history") ? options.history : reactRouterHistory);
```

Comment:
as discussed, the history created in this file is used only if:
1. `options.history` is `true`
2. `options.history` is not `defined`
Otherwise,
1. if it's not `falsy`, assume it's a valid history object and use it with `Router`
2. finally use `BrowserRouter`

Reply:
no double `?:` please